lib/honeybadger/insights/tesla.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Honeybadger.Insights.Tesla do
+  @moduledoc false
+
+  use Honeybadger.Insights.Base
+
+  @required_dependencies [Tesla]
+
+  @telemetry_events [
+    "tesla.request.stop",
+    "tesla.request.exception"
+  ]
+
+  def extract_metadata(meta, _name) do
+    env = Map.get(meta, :env, %{})
+
+    %{
+      method: env.method |> to_string() |> String.upcase(),
+      url: env.url,
```

Comment:
In the [ruby implementation](https://github.com/honeybadger-io/honeybadger-ruby/blob/master/lib/honeybadger/plugins/net_http.rb#L46), we default to recording the hostname only. We enable the full url by setting `net_http.insights.full_url` to true. Is it possible to do that here?

Reply:
Yeah, we can add that.

Reply:
https://github.com/honeybadger-io/honeybadger-elixir/pull/591/commits/d83362d2609e4abab35f57aa8946b1de022a3852